CRM/CiviAwards/DAO/AwardDetail.php

```diff
@@ -197,7 +196,6 @@ public static function &fields() {
           'entity' => 'AwardDetail',
           'bao' => 'CRM_CiviAwards_DAO_AwardDetail',
           'localizable' => 0,
-          'FKClassName' => 'CRM_Core_DAO_UFGroup',
```

Comment:
Can you check why this is removed? The profile ID is actually a foreign key to the `uf_group` table. I think the XML file must have gotten messed up in some way for the regeneration to remove this. Can you please exclude this change.